charts/common/templates/hpa.yaml

```diff
@@ -1,7 +1,9 @@
 {{- /* Rules */}}
 {{- $env := .Values.env | required ".Values.common.env is required." -}}
 
-{{- if (or (eq "prd" .Values.env) .Values.container.maxReplicas) }}
+{{- if .Values.container.forceReplicas }}
```

Comment:
Would it be more natural to use nested ifs (if not .Values.container.forceReplicas) than this empty block?